src/__test__/redux/reducers/experimentsReducer.test.js

```diff
@@ -37,6 +51,40 @@ describe('experimentsReducer', () => {
     }),
   ).toEqual(initialState));
 
+  it('Loads an experiment correct', () => {
+    const newState = experimentsReducer(initialState, {
```

Comment:
I would add another experiment, just to make sure that we test that the reducer also behaves properly on  a non-initial state (specially since the code in `src/redux/reducers/experiments/experimentsLoaded.js` is a bit involved)

Reply:
Added